packages/p2p/src/components/orders/order-details/order-details-footer.jsx

```diff
@@ -1,188 +1,335 @@
+                    <Checkbox
+                        name={
+                            (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad)
+                                ? 'seller_not_released'
+                                : 'buyer_not_paid'
+                        }
+                        className='order-details__complain-checkbox'
+                        onChange={event => {
+                            onChange(event.target.name);
+                        }}
+                        value={dispute_reason === 'seller_not_released' || dispute_reason === 'buyer_not_paid'}
+                        label={
+                            (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad)
+                                ? localize('I’ve made full payment, but the seller hasn’t released the funds.')
+                                : localize('I’ve not received any payment.')
+                        }
+                    />
+                    <Checkbox
+                        name='buyer_underpaid'
+                        onChange={event => {
+                            onChange(event.target.name);
+                        }}
+                        value={dispute_reason === 'buyer_underpaid'}
+                        label={
+                            (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad)
+                                ? localize('I wasn’t able to make full payment.')
+                                : localize('I’ve received less than the agreed amount.')
+                        }
+                    />
+                    <Checkbox
+                        name='buyer_overpaid'
+                        onChange={event => {
+                            onChange(event.target.name);
+                        }}
+                        value={dispute_reason === 'buyer_overpaid'}
+                        label={
+                            (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad)
+                                ? localize('I’ve paid more than the agreed amount.')
+                                : localize('I’ve received more than the agreed amount.')
+                        }
+                    />
+                    <div className='order-details__contact-text'>
+                        <Localize
+                            i18n_default_text="If your complaint isn't listed here, please contact our <0>Customer Support<0/>"
+                            components={[
+                                <a
+                                    key={0}
+                                    className='link order-details__popup--danger'
+                                    rel='noopener noreferrer'
+                                    target='_blank'
+                                    href={'mailto:p2p-support@deriv.com'}
+                                />,
+                            ]}
+                        />
+                        {localize(' team.')}
```

Comment:
Can we add this to the `i18n_default_text` string instead?

Reply:
I tried that but it becomes part of the link. Maybe I'm doing something wrong. I'll discuss this with you 😅